echoshader/dev/echo_curtain.py

```diff
@@ -0,0 +1,68 @@
+"""echo_curtains.py module
+
+to get 3D curtains
+"""
+
+import numpy
+import panel
+import pyvista
+
+panel.extension("vtk")
+
+
+def plot_curtain(MVBS_ds, cmp="jet", ratio=0.001):
+    """Get an echogram and histogram with control widgets
+
+    Parameters
+    ----------
+    MVBS : xr.Dataset
+        MVBS_ds data set or ds_Sv data array with a specific frequency
+        like MVBS_ds.sel(channel='GPT  18 kHz 009072058c8d 1-1 ES18-11')
+
+    Returns
+    -------
+    type: None
+
+    describe:
+        ctreate a pyVista structure 'grid' and 'curtain'
+        call show() to display the 3D curtain
```

Comment:
```suggestion
    None
        ctreate a pyVista structure 'grid' and 'curtain'
        call show() to display the 3D curtain
```

Based on numpy docstring specification, what you have is an incorrect way to document what a function returns. See https://numpydoc.readthedocs.io/en/latest/format.html#returns